src/main/java/pl/ostek/internet_chat/service/MessageService.java

```diff
@@ -14,9 +15,9 @@
     public void sendMessage(Message message) {
         if (message == null)
             throw new MessageServiceException("Message object should not be null!");
-        if (message.getReceiverId() == null || message.getReceiverId().equals(""))
+        if (StringUtils.isBlank(message.getReceiverId()))
```

Comment:
Utilsy z Commons Lang lub Guavy są na tyle popularne w projektach, że nawet pomija się w ich wywołaniu klasę zastępując ją statycznym importem. 
Może czasem wydawać się, że może to wprowadzać w błąd np. metoda isEmpty jest w StringUtils oraz w CollectionUtils, ale kluczowe jest zrozumiałość kodu np.
if (isEmpty(zmienna) ) ...
Nie tyle powinno być istotne z jakiej klasy, czy nawet biblioteki pochodzi ta metoda, ale "intuicyjne" zrozumienie patrząc na taki kod, co tu się odbywa. Jeśli czytelna jest sama metoda, to pominięcie prefiksu klasy, to też mniej do "ogarniania" wzrokiem.